src/journeys_utils.js

```diff
@@ -908,3 +908,55 @@ journeys_utils.setJourneyLinkData = function(linkData) {
 	}
 	journeys_utils.journeyLinkData = data;
 };
+
+journeys_utils.getValueForKeyInBranchViewData = function(key) {
+	if(!journeys_utils){
+		return false;
+	}
+
+	if(!journeys_utils.branch){
+		return false;
+	}
+
+	if(!journeys_utils.branch._branchViewData){
+		return false;
+	}
+
+	if(!journeys_utils.branch._branchViewData.data){
+		return false;
+	}
+
+	return journeys_utils.branch._branchViewData.data[key];
+};
+
+journeys_utils.hasJourneyCtaLink = function () {
+	if(!journeys_utils.getValueForKeyInBranchViewData('$journeys_cta')){
+		return false;
+	}
+
+	return journeys_utils.getBranchViewDataItemOrUndefined('$journeys_cta').length > 0;
+};
+
+journeys_utils.getBranchViewDataItemOrUndefined = function(name){
+	if(journeys_utils.getValueForKeyInBranchViewData(name)){
+		return journeys_utils.branch._branchViewData.data[name];
+	}
+	 return undefined;
+};
+
+journeys_utils.getJourneyCtaLink = function () {
+	return journeys_utils.getBranchViewDataItemOrUndefined('$journeys_cta');
```

Comment:
why not just call getValueForKeyInBranchViewData('$journeys_cta')?

Reply:
Try to increase the readability of the code.